src/Uno.UWP/Storage/Helpers/StorageFileHelper.cs

```diff
@@ -0,0 +1,14 @@
+﻿#nullable enable
+using System.Threading.Tasks;
+
+namespace Windows.Storage.Helpers;
+
+public partial class StorageFileHelper
+{
+	/// <summary>
+	/// Determines if a file exists within application package
+	/// </summary>
+	/// <param name="fileName">relative file path</param>
+	/// <returns>A task that will complete with a result of true if file exists, otherwise with a result of false.</returns>
+	public static async Task<bool> Exists(string fileName) => await FileExistsInPackage(fileName);
```

Comment:
@jeromelaban what's the standard within Uno for async methods - should this be ExistsAsync?

Reply:
Also, should this be made more specific to storage within the application package, for example ExistsInPackage?

Reply:
Naming is in an odd position here, mostly because WinRT does not use that convention. since we're extending WinRT, I'd go on the side of not having the Async suffix.

Reply:
@jeromelaban happy to leave off the Async suffix. I think changing the name to ExistsInPackage would be good. Thoughts?

Reply:
Changing to`ExistsInPackage` is good indeed.